webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -51,7 +51,9 @@ const InListName = 'in';
 
 export default class SidebarRight extends React.PureComponent {
     static propTypes = {
-        allIssues: PropTypes.arrayOf(PropTypes.object),
+        myIssues: PropTypes.array,
```

Comment:
```suggestion
        myIssues: PropTypes.array.isRequired,
```

Same for the two lines below this one. (GitHub isn't letting me comment on multiple lines for some reason)

![image](https://github.com/mattermost/mattermost-plugin-todo/assets/6913320/3e53fb8d-e7fc-46e0-86a0-c7088160e321)